source/assets/sass/layout/_grid-system.scss

```diff
@@ -124,41 +124,41 @@
 
       &--gutters {
         margin: {
-          right: -(functions.sl-px-to-rem(16px));
-          left: -(functions.sl-px-to-rem(16px));
+          right: calc((var(--gutter)) * -1);
```

Comment:
All of these seem to have extra parenthesis? I also noticed a few places where you un-nested margin longhand properties in earlier files - should we be consistent one way or the other?

Reply:
fixed these (by making the negative variables their own variable instead of using calc) and I made a card on making the paddings/margin type of shorthand consistent as it is a mix throughout the site. I think I only updated it in the alert file so I will change that back for now.